libs/dh/shared/data-access-msw/src/lib/mocks/data/messageArchiveSearchResponseLogs.ts

```diff
@@ -16,98 +16,128 @@
  */
 import { SearchResult } from '@energinet-datahub/dh/shared/domain';
 
+const messageId1 = '38374f50-f00c-4e2a-aec1-70d391cade06';
+const messageId2 = '41c8490b-7a7b-45bd-b95b-87659964e7aa';
+const messageId3 = '4f92c84e-d976-43f8-9df2-86d7d3adbf43';
+const messageId4 = '51fa193b-9063-4d39-b8f0-e1966e851c05';
+const messageId5 = '53afd9d8-c558-4a85-957c-26d5b0da5f1f';
+const messageId6 = '5eafaa79-6307-48ea-8380-1f576d7c561f';
+const messageId7 = '81f5667d-155f-44cf-b77a-ac748403defe';
+const messageId8 = '894678e2-7b8a-4b93-a2eb-3498a118aeb1';
+const rejectedMessageId = 'f23b0e28-f173-4348-b83f-b783fa3e5531';
+
 export const messageArchiveSearchResponseLogs: SearchResult = {
   messages: [
     {
-      messageId: '38374f50-f00c-4e2a-aec1-70d391cade06',
+      messageId: messageId1,
       documentType: 'NotifyAggregatedMeasureData',
       createdDate: '2023-06-14T12:10:54.9519045+00:00',
       senderGln: '5790001330552',
       receiverGln: '7080000729821',
+      id: messageId1,
```

Comment:
question: are values for `id` and `messageId` supposed to match? Some of the objects here have the same values, where others don't :)

Reply:
Yes. The "id" and "messageId" will match whenever the archived message is a response from IDE.
But they will most likely not match, whenever the archived message is a request from an actor. Since the messageId will be the identifier of their request and the id will be something we add for uniqueness